app/meetings/detail/-components/meeting-detail-header/component.ts

```diff
@@ -0,0 +1,54 @@
+import { action, computed } from '@ember-decorators/object';
+import { service } from '@ember-decorators/service';
+import Component from '@ember/component';
+import I18N from 'ember-i18n/services/i18n';
+import MeetingModel from 'ember-osf-web/models/meeting';
+import moment from 'moment';
+
+export default class MeetingDetailHeader extends Component {
+    @service i18n!: I18N;
+
+    // Required param
+    meeting!: MeetingModel;
+    // Private property
+    isPanelOpen = false;
+
+    @computed('meeting.{startDate,endDate,location}')
+    get locationAndDateText() {
+        if (!this.meeting) {
+            return null;
+        }
+        const { startDate, endDate, location } = this.meeting;
+        let dateString = null;
+        if (startDate && endDate) {
+            dateString = `${moment(startDate).format('MMMM Do, YYYY')} - ${moment(endDate).format('MMMM Do, YYYY')}`;
+        } else {
+            dateString = `${moment(startDate).format('MMMM Do YYYY')}`;
+        }
+
+        if (location) {
+            return `${location} | ${dateString}`;
+        }
+
+        return dateString;
+    }
```

Comment:
IMHO, this is nicer in the template as:
```hbs
{{if this.meeting.location (concat this.meeting.location ' | ')}}
{{moment-format this.meeting.startDate 'MMMM Do, YYYY'}}
{{if this.meeting.endDate (concat ' - ' (moment-format this.meeting.endDate 'MMMM Do, YYYY')}}
```

Reply:
Also, if we're matching the existing page, the date format should be `MMM DD, YYYY` (see: https://test.osf.io/view/SGGL2018/).